src/app/page/wallet-show-page/wallet-show-page.component.ts

```diff
@@ -7,4 +7,13 @@ import { Component } from '@angular/core';
 })
 export class WalletShowPageComponent {
 
+  public allWallet: string[];
+  constructor() {
+    this.allWallet = [];
+    const totalWallet: number = 10;
+    this.allWallet.push('+');
+    for (let i: number = 0; i < totalWallet; i += 1) {
```

Comment:
use for...of
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
it's a dummy data, added comment to remove it later.